assignments/arrays.js

```diff
@@ -63,32 +63,44 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let carInfo = inventory.find(nameKey => nameKey.id === 33);
+console.log(`Car 33 is a *${carInfo.car_make}* *${carInfo.car_model}*`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.pop();
```

Comment:
you have to be very careful of using methods that MUTATE your array.
by calling the pop function you are also DELETING that last inventory item.
if you `console.log(inventory)` right after this variable declaration you will see there are now only 49 cars listed in your inventory.
.pop() is a method that MUTATES, which means that the original array is NOW CHANGED and EVERYTHING else that runs after this statement will be referring to that altered array which contains only 49 cars.
this page is a handy reference >> https://doesitmutate.xyz/

```
console.log(inventory); //50 cars
let lastCar = inventory.pop();
console.log(`The Last Car in the Inventory is a ${lastCar.car_make} ${lastCar.car_model}`);
console.log(inventory); //49 cars
```